layouts/partials/sidebar.html

```diff
@@ -0,0 +1,43 @@
+<aside>
+    <div class="row top-buffer">
+
+      {{ if .Site.Params.recentPosts }}
+        {{ if and (.Site.Params.recentPosts) (.Site.Params.tags) }}
+          <div class="col-xs-12 col-sm-8 col-md-8">
+        {{ else }}
+          <div class="col-xs-12 col-sm-12 col-md-12">
+        {{ end }}
+              <div class="panel panel-default">
+                  <div class="panel-heading">
+                      <h2 class="panel-title">Recent Posts</h2>
+                  </div>
+                  <div class="list-group">
+                      {{range first .Site.Params.recentPostDisplayCount .Site.Pages}}
+                          <a href="{{.RelPermalink}}" class="list-group-item">{{ .Title }}</a>
+                      {{ end }}
+                  </div>
+              </div>
+          </div>
+      {{ end }}
+
+      {{ if .Site.Params.tags }}
+        <div class="col-xs-12 col-sm-4 col-md-4">
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <h2 class="panel-title">Tags</h2>
+                </div>
+                <div class="list-group">
+                    {{ $baseurl := .Site.BaseURL }}
+                    {{ range $name, $value := .Site.Taxonomies.tags }}
+                    <a href="{{ $baseurl }}/tags/{{ $name | urlize }}" class="list-group-item">
```

Comment:
This style breaks on S3 (No such key) if your baseurl already ends in "/" - which I need because the rest of the CSS breaks if I don't have a trailing "/"

Reply:
Sorry to necro this post but just noticed it now when I ran into the issue porting my site to run on S3. I wish I saw this earlier since it would have saved me a bit of trouble. Anyhow I agree with your suggestion and have pushed up the fix.